src/foundation/colors/theme.ts

```diff
@@ -0,0 +1,161 @@
+// * Theme
```

Comment:
Any plan on using these?

Reply:
Maybe in the very distant future...? Currently, it is simply commented out to check the key values.